### ISSUE TITLE
Enhanced Account Icons

### ISSUE THREAD

**Issue by suprunchuk** [thunderbird/thunderbird-android#8590]
### Checklist

- [x] I have used the search function to see if someone else has already submitted the same bug report.
- [x] I will describe the problem with as much detail as possible.

### App

Thunderbird for Android

### App version

8.1

### Where did you get the app from?

Google Play

### Android version

15

### Device model

Pixel 7

### Steps to reproduce

Issue:
Currently, the Thunderbird Android app displays account icons with only two characters, making it difficult to distinguish between multiple email accounts. This can be particularly inconvenient when switching between accounts, as users cannot easily identify which account is which without selecting it.

Proposed Solution:
Enhance the account icons to include more identifiable information, such as:

Full Email Address Display: Show the full email address or a more significant portion of it.
Customizable Icons: Allow users to set custom icons or avatars for each account.
Benefits:

Improved Usability: Users can quickly identify and switch between accounts without confusion.
Enhanced User Experience: Customizable icons and color coding add a personal touch and make the app more user-friendly.
Reduced Frustration: Users will spend less time trying to figure out which account they are selecting, leading to a smoother email management experience.
Implementation Steps:

Update Icon Display: Modify the account icon display to show more than two characters, ideally the full email address or a significant portion of it.
Add Customization Options: Introduce a feature that allows users to upload or select custom icons for their accounts.

### Expected behavior

Proposed Display: "[example@gmail.com](mailto:example@gmail.com)" or a custom icon with a unique color.

### Actual behavior

Current Display: "GM" for an account with the email address "[example@gmail.com](mailto:example@gmail.com)".

### Logs

![Image](https://github.com/user-attachments/assets/0da3eaca-c938-4c31-9fb3-bc950b9274eb)

**Comment by dabear2** [comment]
same

**Comment by KharosFox** [comment]
+1

This is unusable for someone with many accounts under the same name.

**Comment by Jilas** [comment]
Using the display name is pointless with similar accounts. 

A previous version used actual icons pulled in (I think from the address book).  This worked well much better to identify an account Vs two initials we have now. 

There is already the custom colour, so can we just have an option to import an icon plus for those that want it, to set the custom initials Vs just assuming the display name is unique.

**Comment by syborgxx** [comment]
I agree. There should be an option to display the whole account name as in prior versions of K9 mail.

**Comment by kolAflash** [comment]
related:
- #8238
- #8381
